dcm4che-core/src/main/java/org/dcm4che3/io/DicomInputStream.java

```diff
@@ -107,12 +107,12 @@
     private byte[] byteBuf;
     private int allocateLimit = DEF_ALLOCATE_LIMIT;
     private String uri;
-    private String tsuid;
+    protected String tsuid;
     private byte[] preamble;
     private Attributes fileMetaInformation;
     private boolean hasfmi;
-    private boolean bigEndian;
-    private boolean explicitVR;
+    protected boolean bigEndian;
+    protected boolean explicitVR;
```

Comment:
Wouldn't it be better to add a public or protected setter instead of changing these field's access level?

Reply:
This is not meant to be a Java enterprise Bean (with get/set). These fields are only used internally so I think its acceptable to access them this way.